### DISCUSSION TITLE
FLIPPER WON'T UPDATE

### DISCUSSION THREAD

**Discussion by MYSTICBLAZE** [community/community#70335] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

[RPC] Cannot match message with id 14
[RPC] Cannot match message with id 14
[RPC] Cannot match message with id 14
[RPC] Cannot match message with id 14
[RPC] Cannot match message with id 14
[RPC] Cannot match message with id 14
[RPC] Cannot match message with id 14
[RPC] Cannot match message with id 14
[RPC] Cannot match message with id 14
[RPC] Cannot match message with id 14
[RPC] Cannot match message with id 14
[RPC] Cannot match message with id 14
[RPC] Cannot match message with id 14
[RPC] Cannot match message with id 14

**Comment by MYSTICBLAZE** [comment]
Everytime I attemp to update. I get the same message.
But different id number[RPC] Cannot match message with id 30
[RPC] Cannot match message with id 30
[RPC] Cannot match message with id 30
[RPC] Cannot match message with id 30
[RPC] Cannot match message with id 30
[RPC] Cannot match message with id 30
[UTL] Start Update @Chama ERROR: Manifest folder not found
[DEV] Full Update @Chama ERROR: Manifest folder not found

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Reply by MYSTICBLAZE** [reply]
That's the problem. I have PTSD and don't remember.I do remember that I fixed this problem before .And it was in my FLIPPER folder. But I accidentally quick formatted my external hard drive.
And now, trying to recover through as many possible softwares I can find.
I used to be a master programmer.From DIRECT TV CARDS to DISH NETWORK.Now. I've been out of the loop for so long.It all looks like a different language.
I understand repositories from my prograsmming FIRE STICKS with KODI...And that's about it.
